packaging/mingw-w64/configure-build-linux-mingw.sh

```diff
@@ -0,0 +1,14 @@
+cmake -DCMAKE_TOOLCHAIN_FILE=./packaging/mingw-w64/mingw_toolchain.cmake \
+      -DCMAKE_INSTALL_PREFIX=$PWD/dest/ \
+      -DMINGW_BUILD:BOOL=ON \
+      -DWITHOUT_WIN32_CONFIG:BOOL=ON \
+      -DRDKAFKA_BUILD_EXAMPLES:BOOL=ON \
+      -DRDKAFKA_BUILD_TESTS:BOOL=ON \
+      -DWITH_LIBDL:BOOL=OFF \
+      -DWITH_PLUGINS:BOOL=OFF \
+      -DWITH_SASL:BOOL=OFF \
+      -DWITH_SSL:BOOL=OFF \
```

Comment:
As started previously, this build without SSL (and zlib) will not be widely usable.

Reply:
Maybe I should just get rid of the cross-compiled build? It was mainly intended to check if the code still builds with a Linux host (as I'm currently doing on my project) but if official MinGW builds start becoming available then I don't actually need that any more.